### ISSUE TITLE
Minimum votes required to implement accessibility  issue should be far less than normal issues.

### ISSUE THREAD

**Issue by balajip881** [microsoft/vscode#184475]
There are  a lot fewer number of assistive technology users  compared to normal users. So we should try to give level playing feild to assistive technology users by reducing the number of votes required to move it to implementation.There are less than 5% people only use assistive technologies.

**Comment by meganrogge** [comment]
Thanks for the issue - we already do this, pls provide an example if you feel that's not the case

**Comment by balajip881** [comment]
I have seen the issue triaging docs but there is mention of upvoting policy link is no mention of numberof votes required specifically for accessibility issues.
Please treat Accessibility issue  as important  as UI issue so that accessibility users also can benifit.
Please treat accessibility issue as a bug so that community can implement quicker.thanks.

**Comment by meganrogge** [comment]
Yes, though it's not documented those are already our current practice.
